homeassistant/components/philips_js/switch.py

```diff
@@ -93,13 +96,13 @@ def __init__(
 
         super().__init__(coordinator)
 
-        self._attr_name = f"{coordinator.system['name']} Ambilight+Hue"
         self._attr_icon = "mdi:television-ambient-light"
         self._attr_unique_id = f"{coordinator.unique_id}_ambi_hue"
         self._attr_device_info = DeviceInfo(
             identifiers={
                 (DOMAIN, coordinator.unique_id),
-            }
+            },
+            name=coordinator.system["name"],
```

Comment:
I dont think this is needed, but i suppose it doesnt hurt.

Reply:
I mean you're already registering a device, why not name it

Reply:
The device data will come from more central entities. It would make sense to add a standard property on the coordinator, so they all returned the same object.

Reply:
Ahh, I see. Another option would be a superclass.

Reply:
I made #96756 to add the superclass to the integration